<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labeWelcome">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>471</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Chào mừng {username} đến với phần mềm quản lý đoàn viên</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQLDV">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Quản lý đoàn viên</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQLHD">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Quản lý hoạt động</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnQLDP">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Quản lý đoàn phí</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBCTK">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>290</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Báo cáo và thống kê</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
